src/test.js

```diff
@@ -485,15 +485,15 @@ function synchronize( callback, priority ) {
 
 // Place previously failed tests on a queue priority line, respecting the order they get assigned.
 function priorityFill( callback ) {
-	var queue, prioritizedQueue;
+	var highPrioritized,
+		lowPrioritized;
```

Comment:
we may keep the both unassigned names on the same line, as: `var highPrioritized, lowPrioritized;`